### PR TITLE
fix: mediaservices model validation errors

### PR DESCRIPTION
**Pull request by nschonni**
- AssetName -> assetName
- StreamingPolicyName -> streamingPolicyName
- StreamingLocatorId -> streamingLocatorId
- ContentKeys -> contentKeys
- EndTime -> endTime
- StartTime -> startTime
- Properties -> properties
- Id -> id
- Value -> value
- LabelReferenceInStreamingPolicy -> labelReferenceInStreamingPolicy
- "tags" to object
- Remove empty body for RESPONSE_SCHEMA_NOT_IN_SPEC errors

### CODE REVIEW

**Comment by NullMDR** [comment]
@openapibot sdkautomation rebuild

**Review by BrianBlum — APPROVED** [review]
These changes look good.

**Comment by BrianBlum** [comment]
@nschonni , thanks for the cleanup

File: specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/examples/liveevent-delete.json [hunk 1]
```diff
@@ -6,7 +6,7 @@
     "liveEventName": "myLiveEvent1",
     "api-version": "2018-07-01",
     "parameters": {
-      "Id": "/subscriptions/0a6ec948-5a62-437d-b9df-934dc7c1b722/resourceGroups/mediaresources/providers/Microsoft.Media/mediaservices/slitestmedia10/liveevents/myLiveEvent1",
+      "id": "/subscriptions/0a6ec948-5a62-437d-b9df-934dc7c1b722/resourceGroups/mediaresources/providers/Microsoft.Media/mediaservices/slitestmedia10/liveevents/myLiveEvent1",
       "Name": "myLiveEvent1",
```
Comment (BrianBlum) [inline specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/examples/liveevent-delete.json]: @nschonni , were these caught by a tool? I was curious why name, type and location weren't flagged. The definition in ..\common.json has lowercase names for the Resource.

Comment (nschonni) [reply specification/mediaservices/resource-manager/Microsoft.Media/stable/2018-07-01/examples/liveevent-delete.json]: Just running OAV across the repository with the latest version. There may be more, but the newer version catches stuff that wasn't likely flagged when the original files were PR'd.
I just PR example changes that clean up some of the new failures, as long as I'm pretty sure that it wasn't a spec issue.

**Review by yungezz — APPROVED** [review]

**Pull request merged**